namespace/digest.go

```diff
@@ -24,7 +27,7 @@ func IntervalDigestFromBytes(nIDLen IDSize, digestBytes []byte) (IntervalDigest,
 	}, nil
 }
 
-func (d IntervalDigest) Min() ID {
+func (d IntervalDigest) Min() ID {// TODO: I don't like that we cannot compare Digests without allocating to much
```

Comment:
Either remove the todo or turn into sth more constructive. What is needed to reduce allocations?
Also, ideally, they would start with `TODO(Wondertan):`

Reply:
Haha, that's an accidental paste from another piece of work. It does not even make sense anymore. Will force push it